pkg/util/k8sutil/k8sutil.go

```diff
@@ -315,12 +315,7 @@ func InClusterConfig() (*rest.Config, error) {
 	return rest.InClusterConfig()
 }
 
-func NewTPRClient() (*rest.RESTClient, error) {
```

Comment:
It's by design to make NewTPRClient() self-contained.
Again, what's the problem to call InClusterConfig() twice?